app/components/MobileNav/MobileNav.tsx

```diff
@@ -11,8 +11,17 @@ export default function MobileNav({ links }: { links: Array<LinkInfo> }) {
         <HamburgerButton showOverlay={() => setShowOverlay(true)} />
     );
     const overlayNav = (
-        <OverlayNav closeOverlay={() => setShowOverlay(false)} links={links} />
+        <OverlayNav
+            showOverlay={showOverlay}

```

Comment:
I like this, giving showOverlay as a prop instead of conditionally rendering it. Means it's always in the DOM and more performant I'm guessing?

Reply:
Just allows the visibility of the overlay nav to be controlled externally from the component. I suppose we could merge overlay and mobile nav into one component?